modules/abstract-eth/src/lib/transferBuilders/baseNFTTransferBuilder.ts

```diff
@@ -38,6 +39,11 @@ export abstract class BaseNFTTransferBuilder {
     throw new InvalidParameterValueError('Invalid expiration time');
   }
 
+  walletVersion(version: number): this {
```

Comment:
Nit: This function scheme sounds like a getter opposed to a setter. Maybe `setWalletVersion`?